Header header
# true if the play was a pass, else it was a run
bool isPass
# if this was a pass, true if it was completed
bool passCompleted
# number of yards gained/lost from the play
int32 yards
# true if this play resulted in a touchdown
bool touchdown
